pkg/controller/upload-controller_test.go

```diff
@@ -498,6 +498,8 @@ var _ = Describe("Update PVC", func() {
 func createUploadReconciler(objects ...runtime.Object) *UploadReconciler {
 	objs := []runtime.Object{}
 	objs = append(objs, objects...)
+	cdiCR := MakeEmptyCDICR()
```

Comment:
I don't think I pointed out all locations where it could be simplified. This is one of the others.